e2e-tests/okta-hosted-login/conf.js

```diff
@@ -0,0 +1,65 @@
+/*!
+ * Copyright (c) 2015-2016, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+/* eslint import/no-unresolved:0, import/no-extraneous-dependencies:0, no-console:0 */
+/* global jasmine */
+const jasmineReporters = require('jasmine-reporters');
+const daemonUtil = require('../tools/daemon-util');
+
+const promises = Promise.all([
+  daemonUtil.startOktaHostedLoginServer()
+]);
+
+const config = {
+  // Update these parameters to match your environment for tests
```

Comment:
@robertjd - User can pass his configuration in this file for login

Reply:
Thanks.  Let's take this one step further and have this read from the environment, e.g.:

```
params: {
  login: {
    username: process.env.USERNAME,
    password: process.env.PASSWORD,
    email: process.env.USERNAME,
  },
},
```

This makes everything clean and pluggable.  Also: can we refactor to only need a username and password?  Is there a reason to test this with a username that's not an email address?

Reply:
Can we name them something more descriptive then user/pass? TEST_USER_EMAIL (or anything really)

Reply:
Yeah TEST_ prefix sounds great

Reply:
FYI: In our Angular, React, and Vue SDKs, we test with `process.env.USERNAME` and `process.env.PASSWORD`.

`TEST_USERNAME` seems too specific for this testing case, while `USERNAME` can be used across our different integrations.

Reply:
I've named them as USERNAME & PASSWORD for now. If there's much opposition to using this, I can update it to user TEST prefix later.